### ISSUE TITLE
Question about loss

### ISSUE THREAD

**Issue by yuyijie1995** [experiencor/keras-yolo3#111]
I have trained my dataset several times using this code ,but I found that the loss_yolo_1 and 2 are already very small like 0.1 but loss_yolo_3 is still very big and hard to decrease .Should I change some default parameters to refine this problem? This is my config file:
![image](https://user-images.githubusercontent.com/30818917/42014230-c02b2246-7ad3-11e8-8852-afea2f925ab8.png)

![image](https://user-images.githubusercontent.com/30818917/42014235-c76b9892-7ad3-11e8-9f45-863a5af6a669.png)

**Comment by experiencor** [comment]
You can change grid_scales. 3 numbers for 3 scales 1, 2, 3.

**Comment by yuyijie1995** [comment]
Thanks a lot ,I will try .

**Comment by Jiyugaoka** [comment]
@yuyijie1995 I meet the same problem on my own dataset, the total is close to 10, no more falling down. And the detection results don' t preform as good as expected. Do you fix this probelm? How it works? Looking forward to your reply.

**Comment by yuyijie1995** [comment]
@Jiyugaoka I changed the original learning rate to 1e-4 and got a smaller loss value

**Comment by Jiyugaoka** [comment]
@yuyijie1995 the original lr is already 1e-4. Do you mean replace it with a smaller one?
If you can explain to me how these [grid_scales, obj_scale, noobj_scale, xywh_scale, class_scale] parameters work, I will be grateful.

**Comment by yuyijie1995** [comment]
I also chenged the 'train_times' to 8. I treid to modify the grid_scales but I didt get better result.

**Comment by yuyijie1995** [comment]
@Jiyugaoka

**Comment by Jiyugaoka** [comment]
@yuyijie1995 Can I add you as a friend on the wechat? So that we can communicate effectivelly.My wechat id is zhangyx-0920.
